### ISSUE TITLE
Add missing WHERE to last UPDATE example on MovR Doc 

### ISSUE THREAD

**Issue by dbist** [cockroachdb/docs#7898]
Re: [MovR](https://cockroachlabs.com/docs//v20.1/movr.html)

## Issue Description
the last example showing the update statement needs a `WHERE` clause, otherwise it fails with 

```sql
root@127.0.0.1:58842/movr> UPDATE rides SET end_address = '75257 David Valley Suite 98';
ERROR: rejected: UPDATE without WHERE clause (sql_safe_updates = true)
SQLSTATE: 01000
```
## Suggested Resolution 

```sql
UPDATE rides SET end_address = '75257 David Valley Suite 98' WHERE id = '51672693-c8f3-4cca-9fac-f59619783ec7';
```

**Comment by irkan-hadi** [comment]
Would like to take this issue and add copy-paste friendly examples with real values

**Comment by ericharmeling** [comment]
> Would like to take this issue and add copy-paste friendly examples with real values

@irkan-hadi 
Thanks for offering your contribution! I think this example just needs an additional `WHERE` clause.

As @dbist suggests:

```sql
UPDATE rides SET end_address = '75257 David Valley Suite 98' WHERE id = '51672693-c8f3-4cca-9fac-f59619783ec7';
```

**Comment by irkan-hadi** [comment]
@ericharmeling ah yes, but his suggestion is a fully executable statement while the others in the official doc are just skeleton statements (to be filled by the user) so I thought it would look out of place unless some fully executable examples are added as well. 

www.cockroachlabs.com/docs/stable/movr.html

**Comment by ericharmeling** [comment]
> @ericharmeling ah yes, but his suggestion is a fully executable statement while the others in the official doc are just skeleton statements (to be filled by the user) so I thought it would look out of place unless some fully executable examples are added as well.
> 
> [www.cockroachlabs.com/docs/stable/movr.html](http://www.cockroachlabs.com/docs/stable/movr.html)

Good point! Feel free to open a PR to our docs repo. I'd be happy to review. :)

**Comment by ericharmeling** [comment]
Fixed by #8520. Thanks @irkan-hadi !
